Kros.KORM/tests/Kros.KORM.Extensions.Api.UnitTests/ConfigurationHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Kros.KORM.Extensions.Api.UnitTests
+{
+    internal class ConfigurationHelper
+    {
+        private static IConfigurationRoot GetConfiguration(string configName)
+            => new ConfigurationBuilder().AddJsonFile($"appsettings.{configName}.json").Build();
+
+        internal static (IConfigurationRoot configuration, IServiceCollection services) CreateHelpers(string configName)
```

Comment:
I am not 100% sure (and I doon't mind it so much), but I think that general practice is to have internal class but the methods are not, because they do not need to be.